### DISCUSSION TITLE
Chinese version of doc needs an update

### DISCUSSION THREAD

**Discussion by Yangthemob** [tailwindlabs/tailwindcss#10101] [Help]
Chinese version of doc is now at v2.2.16, it is not compeletly translated, and also not up to date.
I think we can give the task to China users' community.

**Comment by adamwathan** [comment]
Unfortunately we have absolutely nothing to do with that site so nothing we can do to update it unfortunately :/

**Reply by Yangthemob** [reply]
oh really.......that's pity.Thanks anyway.
